src/_pytest/main.py

```diff
@@ -19,12 +20,15 @@
 from _pytest.runner import collect_one_node
 
 # exitcodes for the command line
-EXIT_OK = 0
-EXIT_TESTSFAILED = 1
-EXIT_INTERRUPTED = 2
-EXIT_INTERNALERROR = 3
-EXIT_USAGEERROR = 4
-EXIT_NOTESTSCOLLECTED = 5
+
+
+class ExitCode(enum.IntEnum):
+    OK = 0
+    TESTS_FAILED = 1
+    INTERRUPTED = 2
```

Comment:
Just occurred to me: is there a consensus about how to name the enumerate members? `test_failed`, `TestsFailed` or `TESTS_FAILED`? I've not seen many enums in the wild so I'm not sure. 🤔

Reply:
Hmm all examples in the `enum` docs show ALL_CAPS, so I think we are fine. 👍

Reply:
using PEP8 for constants

Reply:
basically enum members are constants, thus follow the same rules as constants ^^